HyperVExtension/src/DevSetupEngine/ConfigurationFileHelper.cs

```diff
@@ -323,43 +349,16 @@ private IInputStream StringToStream(string str)
     /// <summary>
     /// Check if the App Installer needs to be updated.
     /// </summary>
-    /// <param name="currentInstallerPackage">Microsoft.DesktopAppInstaller package</param>
+    /// <param name="currentInstallerPackageVersion">Package version</param>
     /// <returns>true if current Microsoft.DesktopAppInstaller package version is less than 1.22.10661.0</returns>
-    private static bool IsAppInstallerUpdateNeeded(Package currentInstallerPackage)
+    private static bool IsAppInstallerUpdateNeeded(PackageVersion currentInstallerPackageVersion)
     {
-        var packageVersion = currentInstallerPackage.Id.Version;
-        const int minMajor = 1;
-        const int minMinor = 22;
-        const int minBuild = 10661;
-
-        if (packageVersion.Major > minMajor)
-        {
-            return false;
-        }
-        else if (packageVersion.Major < minMajor)
-        {
-            return true;
-        }
-
-        if (packageVersion.Minor > minMinor)
-        {
-            return false;
-        }
-        else if (packageVersion.Minor < minMinor)
-        {
-            return true;
-        }
-
-        if (packageVersion.Build > minBuild)
-        {
-            return false;
-        }
-        else if (packageVersion.Build < minBuild)
-        {
-            return true;
-        }
+        return currentInstallerPackageVersion.LessThan(1, 22, 10661, 0);

```

Comment:
Would this version ever need to change?  If so, maybe we can put it in a manifest?

Reply:
No. It's the minimum version we support.